test/e2e/framework/framework_helper.go

```diff
@@ -145,22 +146,24 @@ func ExecuteCmdAndBuildJSONOutput[T PluginInfo | PluginSearch | PluginGroup | Pl
 	if stdErr != nil {
 		stdErrStr = stdErr.String()
 	}
-	if err != nil {
-		log.Errorf(ErrorLogForCommandWithErrStdErrAndStdOut, cmd, err.Error(), stdErr.String(), out.String())
-		return nil, outStr, stdErrStr, err
-	}
-	jsonStr := out.String()
+
 	var list []*T
-	err = json.Unmarshal([]byte(jsonStr), &list)
-	if err != nil {
-		log.Errorf(FailedToConstructJSONNodeFromOutputAndErrInfo, jsonStr, err.Error())
-		log.Errorf("trying with yaml unmarshal")
-		// try with yaml format unmarshal
-		err2 := yaml.Unmarshal([]byte(jsonStr), &list)
-		if err2 != nil {
-			return nil, outStr, stdErrStr, errors.Wrapf(err, FailedToConstructJSONNodeFromOutput, jsonStr)
+	if outStr != "" {
```

Comment:
So if there is output on stdout we ignore the err of line 140?

Reply:
Good catch, we should not ignore, i have updated the code.